ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -150,7 +151,27 @@ class NavigationControllerDemoController: DemoController {
 
 // MARK: - NavigationControllerDemoController: UIGestureRecognizerDelegate
 
-extension NavigationControllerDemoController: UIGestureRecognizerDelegate {
+extension NavigationControllerDemoController: UIGestureRecognizerDelegate, SearchBarDelegate {
+
+       func searchBarDidBeginEditing(_ searchBar: SearchBar) {
+               searchBar.progressSpinner.stopAnimating()
+               searchBar.progressSpinner.isHidden = true
```

Comment:
Or is this the property to make the spinner optional as @Victa mentioned [here](https://ghe-us.microsoft.com/Microsoft-Design-System-Studio/Design-Toolkit-iOS/issues/1#issuecomment-332749)?
("I suggest to make it optional.")

If that's the case, I think there should be a toggle switch to set the spinner on/off instead of demoing in the delegate. Setting these here is redundant with the hidesWhenStopped property of the ActivityIndicatorView.

Reply:
removed. Thanks

---
In reply to: [468181951](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r468181951) [](ancestors = 468181951)

Reply:
On the demo view there is no place to put an extra switch element. Hence, tried to give search start and stop as the progress spinner's start and stop entry point which will mostly be the case in real use case.
Let me know your thoughts.

---
In reply to: [468195652](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r468195652) [](ancestors = 468195652)

Reply:
Thanks for adding the switch to showcase the optional spinner! 🥇

Reply:
Thanks!

---
In reply to: [470779096](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470779096) [](ancestors = 470779096)